### PR TITLE
ZBUG-3378: Removed clientuploader zimlet and extension

### PR DESCRIPTION
**Pull request by gautamdg**
Removing client uploader zimlet and extension from code.

### CODE REVIEW

**Review by silentsakky — APPROVED** [review]

**Comment by umagmrit** [comment]
@deepakzimb are we using https://github.com/Zimbra/zm-mailbox/blob/develop/store/src/java/com/zimbra/cs/account/ldap/upgrade/BUG_76427.java#L52 ? Do we need to remove this also ?

**Review by umagmrit — APPROVED** [review]

**Comment by umagmrit** [comment]
> @deepakzimb are we using https://github.com/Zimbra/zm-mailbox/blob/develop/store/src/java/com/zimbra/cs/account/ldap/upgrade/BUG_76427.java#L52 ? Do we need to remove this also ?

got it https://github.com/Zimbra/zm-mailbox/pull/1478

**Review by suchitgupta11 — APPROVED** [review]

**Review by vishnukantjangid — APPROVED** [review]

**Review by shrutig0510 — APPROVED** [review]

File: instructions/FOSS_staging_list.pl [hunk 1]
```diff
@@ -128,14 +128,6 @@
          SysExec("cp -f build/zimlet/*.zip $CFG{BUILD_DIR}/zm-certificate-manager-admin-zimlet/build/zimlet");
       },
    },
-   {
-      "dir"         => "zm-clientuploader-admin-zimlet",
```
Comment (silentsakky) [inline instructions/FOSS_staging_list.pl:132]: we have removed this from here but we haven't written circleci workflows to generate these extension/zimlet so how are we planning to generate artefacts for release?

Comment (gautamdg) [reply instructions/FOSS_staging_list.pl:132]: Sure, we will add ci circle YML file for packages.

Comment (gautamdg) [reply instructions/FOSS_staging_list.pl:132]: @umagmrit Please take care it.

**Pull request merged**